src/components/PeopleTable.tsx

```diff
@@ -1,4 +1,28 @@
-export const PeopleTable = () => {
+import React, { useState } from 'react';
+import { Person } from '../types';
+import { SortType, PropName } from './enum';
+
+interface Props {
+  people: Person[],
+  handleSort: (value: string) => void,
+  sortOrder: SortType,
+  propName: string,
+}
+
+export const PeopleTable: React.FC<Props> = ({
+  people,
+  handleSort,
+  sortOrder,
+  propName,
+}) => {
+  const [selectedName, setSelectedName] = useState('');
+  const handleSelection
+  = (event: React.MouseEvent<HTMLAnchorElement>, name: string) => {
+    event.preventDefault();
+
+    setSelectedName(name);
+  };
+
   return (
```

Comment:
Consider using a more descriptive name for the 'propName' prop, such as 'sortField'.